### PR TITLE
Update contributing.md with formatting guidance

### PR DESCRIPTION
**Pull request by bamurtaugh**
As I find myself resharing the guidelines we determined in https://github.com/devcontainers/devcontainers.github.io/pull/307, let's officially document them.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -48,3 +59,9 @@ We use the following [labels](https://github.com/devcontainers/spec/labels):
 - `finalization`: Proposals we intend to make part of the spec.
 
 [Milestones](https://github.com/devcontainers/spec/milestones) use a "month year" pattern (i.e. January 2022). If a finalized proposal is added to a milestone, it is intended to be merged during that milestone.
+
+## Community Engagement
+There are several additional options to engage with the dev container community, such as asking questions, providing feedback, or engaging on how your team may use or contribute to dev containers:
+- [GitHub Discussions](https://github.com/devcontainers/spec/discussions): This is a great opportunity to connect with the community and maintainers of this project, without the requirement of contributing a change to the actual spec (which we see more in issues and PRs)
+- [Community Slack channel](https://aka.ms/dev-container-community): This is a great opportunity to connect with the community and maintainers
+- You can always check out the issues and PRs (and contribute new ones) across the repos in the [Dev Containers GitHub org](https://github.com/devcontainers/devcontainers.github.io/edit/gh-pages/contributing.md) too!
```
Comment (samruddhikhandale) [inline contributing.md]:
```suggestion
- You can always check out the issues and PRs (and contribute new ones) across the repos in the [Dev Containers GitHub org](https://github.com/devcontainers) too!

```

Wondering, if it's a typo or intentional? 🤔

**Review by samruddhikhandale — DISMISSED** [review]
This is awesome! Thank you! 👏 

Left one comment, but otherwise looks great.

File: contributing.md (hunk 1 above)
Comment (bamurtaugh) [reply contributing.md]: Typo 😄 thank you for catching!

**Review by samruddhikhandale — APPROVED** [review]
🚢  it!

**Pull request merged**